datafusion/optimizer/src/simplify_expressions/expr_simplifier.rs

```diff
@@ -1399,6 +1399,18 @@ impl<S: SimplifyInfo> TreeNodeRewriter for Simplifier<'_, S> {
             // Rules for Case
             //
 
+            // CASE WHEN true THEN A ... END --> A
+            Expr::Case(Case {
+                expr: None,
+                when_then_expr,
+                else_expr: _,
+            }) if !when_then_expr.is_empty()
+                && matches!(when_then_expr[0].0.as_ref(), 
+                    Expr::Literal(ScalarValue::Boolean(Some(true)), _)) =>
```

Comment:
can we please use `is_true` to follow the pattern in the rest of this method?

https://github.com/apache/datafusion/blob/4b9a468cc1949062cf3cd8685ba8ced377fd212e/datafusion/optimizer/src/simplify_expressions/utils.rs#L193-L198